### ISSUE TITLE
./src/OVAL/probes/unix/file.c and BSD

### ISSUE THREAD

**Issue by jlcharton** [OpenSCAP/openscap#844]
Hi,

I'm surprised by the code of the 3 functions get_atime, get_mtime and get_ctime for BSD.
One have the following:
```#if defined(OS_FREEBSD)
# if (__STDC_VERSION__ >= 199901L)
		(uint64_t) st->st_atimespec.tv_sec
# else
		(uint64_t) st->st_atimespec.tv_sec
# endif
```
and the same thing for st_ctimespec and st_mtimespec. As you can see, the code is the same for the #if and #else statements. Does the goal of this test is to use either the st_atimespec field (struct timespec) or the traditional st_atime field (time_t)?

**Comment by jan-cerny** [comment]
This is really funny. The code doesn't make any sense .

I have found that the statements used to be different before. See commit https://github.com/OpenSCAP/openscap/commit/35b78b

Now I suggest removing the #if-#else-#endif.

**Comment by jlcharton** [comment]
Fixed by 4d7c745 (PR #846)
